src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private static Color defaultColor = Color.BLACK;
+    private static int defaultLength = 5;
```

Comment:
default logic will be applied only for Circle, It is not needed in other figures